ansible/files/configuration/configuration.json

```diff
@@ -20,9 +20,11 @@
     "Properties": {
       "spark.local.dir": "/mnt,/mnt1",
       "spark.driver.maxResultSize": "4g",
+      "spark.driver.memory": "4g",
```

Comment:
I sent a support request to AWS to get clarification on 'maximizeResourceAllocation'. This seems okay to do.